supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -972,10 +974,11 @@ Use the name of the tab when you refer to it, for example, "the *Storage* tab".
 *See also*:
 
 // Fuse: Added "In Red Hat Fuse,"
+// Fuse: Removed the sentence about Fuse tooling (Breda)
 [discrete]
 [[routing-context]]
 ==== image:images/yes.png[yes] routing context (noun)
-*Description*: In Red Hat Fuse, a routing context specifies the routing rules for a Camel application. Among other things, routing rules specify the source and type of input, how to process it, and where to send the output when processing is done. In Fuse tooling, the routing context is provided in a `.xml` file, the name of which depends on the configuration framework used. For Spring-based projects, the default name of the routing context file is `camelContext.xml`. For Blueprint-based projects, the default name of the routing context file is `blueprint.xml`.
+*Description*: In Red Hat Fuse, a routing context specifies the routing rules for a Camel application. Among other things, routing rules specify the source and type of input, how to process it, and where to send the output when processing is done. For Spring-based projects, the default name of the routing context file is `camelContext.xml`. For Blueprint-based projects, the default name of the routing context file is `blueprint.xml`.
```

Comment:
Consider using bullet points to avoid the passives:

Routing rules provide the following tasks:

- Specify the source and type of input.
- Determine the mechanism for processing any input.
- Direct any output to a source.

Reply:
Sorry @dfitzmau -- I actually think the original text is better than the suggested rewrite in this case! 
@MelissaFlinn @bergerhoffer, what do you think?

Reply:
We actually usually try to avoid doing complex descriptions (I know we have some, but :shrug:). There are some tools out there that parse our glossary entries (Vale for example I think is, or is trying to). So I'd recommend keeping how it is now too.

Reply:
NP. I'll mark this comment as resolved. Thanks all.